core/eolearn/core/utilities.py

```diff
@@ -48,43 +48,60 @@ class FeatureParser:
     can be obtained by iterating over an instance of the class. An `EOPatch` is given as a parameter of the generator.
 
     General guidelines:
-        - Almost all `EOTask`s have take as a parameter some information about features. The purpose of this class is
-        to unite and generalize parsing of such parameter over entire eo-learn package
-        - The idea for this class is that it should support more or less any logical way how to describe a collection
-        of features.
-        - Parameter `...` is used as a contextual clue. In the supported formats it is used to describe the most obvious
-        way how to specify certain parts of feature collection.
-        - Supports formats defined with lists, tuples, sets and dictionaries.
+
+    - Almost all EOTasks have take as a parameter some information about features. The purpose of this class is
```

Comment:
Either "Almost all EOTasks have" or "Almost all EOTasks take", but not "have take".

Reply:
I rephrased the entire sentance. Should be better now.